src/main/java/pl/piekoszek/gorskimatches/equation/SolutionToQuizzesMapper.java

```diff
@@ -0,0 +1,54 @@
+package pl.piekoszek.gorskimatches.equation;
+
+import java.util.*;
+
+public class SolutionToQuizzesMapper {
+    private final CharacterChangeInString characterChangeInString = new CharacterChangeInString();
+    Map<Character, List<Character>> numberChangeableToOther = new HashMap<Character, List<Character>>();
+    List<String> quizEquationsSingleNumber = new ArrayList<>();
+    Map<String, HashSet<String>> quizesAndSolutions = new HashMap<String, HashSet<String>>();
```

Comment:
```
private final Map<Character, List<Character>> numberChangeableToOther = new Map<Character, List<>>();
```
User interface in places other than new instance creation (`Map` instead of `HashMap`, `List` instead of `ArrayList` etc.)

Reply:
ok